HelpSource/Classes/Pkey.schelp

```diff
@@ -31,4 +31,28 @@ p.next(())	// for Pbind, must pass in a default event even if empty
 ( 'a': 5, 'b': 10 )
 ( 'a': 4, 'b': 8 )
 ( 'a': 2, 'b': 4 )
+
+// note: to check if a Pkey's result /equals/ another value,
+// be sure to use |==|
```

Comment:
is there a reason why you use a random process? It is perhaps easier to understand and demonstrate if it were, say, a `Pseries`.